test/AirDrop.test.js

```diff
@@ -0,0 +1,327 @@
+            });
+
+            it("call 'dropTokens' function - false (signed transaction expired)", async () => {
+                const currentBlock = await web3.eth.getBlock("latest");
+                let time = new BN(currentBlock.timestamp);
+                let newTime = time.add(new BN(0)).toString();
+                let sign = await signVerification(web3.utils.toWei("1", "ether"), newTime, acc2, owner);
+                await expectRevert(instanceAirDrop.dropTokens(acc2, web3.utils.toWei("1", "ether"), newTime, sign.v, sign.r, sign.s), "Error : signed transaction expired");
+            });
+
+            it("call 'dropTokens' function - false (invalid signature not owner)", async () => {
+                const currentBlock = await web3.eth.getBlock("latest");
+                let time = new BN(currentBlock.timestamp);
+                let newTime = time.add(new BN(60)).toString();
+                let sign = await signVerification(web3.utils.toWei("1", "ether"), newTime, acc2, acc2);
+                await expectRevert(instanceAirDrop.dropTokens(acc2, web3.utils.toWei("1", "ether"), newTime, sign.v, sign.r, sign.s), "invalid signature");
+            });
+
+        });
+
+        describe("dropTokens - done", async () => {
+            it("call 'dropTokens' function - done", async () => {
+                const currentBlock = await web3.eth.getBlock("latest");
+                let time = new BN(currentBlock.timestamp);
+                let newTime = time.add(new BN(60)).toString();
+                let sign = await signVerification(web3.utils.toWei("1", "ether"), newTime, acc2, owner);
+                let tx = await instanceAirDrop.dropTokens(acc2, web3.utils.toWei("1", "ether"), newTime, sign.v, sign.r, sign.s);
+                await expectEvent(tx, "DropTokens", { recepient: acc2, amount: web3.utils.toWei("1", "ether"), deadline: newTime });
+            });
+        });
+    });
+
+    describe("claimToken", async () => {
+        it("call 'claimToken' function - false (No availabale tokens to claim)", async () => {
+            expect(await web3.eth.getBalance(instanceAirDrop.address)).to.be.bignumber.equal(ether('10'));
+            await expectRevert(instanceAirDrop.claimToken({ from: owner }), "Error : No available tokens to claim");
+            expect(await web3.eth.getBalance(instanceAirDrop.address)).to.be.bignumber.equal(ether('10'));
+        });
+
+        it("call 'claimToken' function - done", async () => {
+            expect(await instanceToken.balanceOf(acc2)).to.be.bignumber.equal(ether('0'));
+            expect(await instanceToken.balanceOf(instanceAirDrop.address)).to.be.bignumber.equal(ether('5'));
+            await instanceAirDrop.claimToken({ from: acc2 });
+            expect(await instanceToken.balanceOf(instanceAirDrop.address)).to.be.bignumber.equal(ether('4'));
+            expect(await instanceToken.balanceOf(acc2)).to.be.bignumber.equal(ether('1'));
```

Comment:
should check event, should check also if acc2 call claimToken again after last claim.

should check balanceOfTokens, balanceOfEthers chang after transaction